.checkstyle/checkstyle.xml

```diff
@@ -101,7 +101,7 @@
 
         <module name="ClassFanOutComplexity">
             <!-- default is 20 -->
-            <property name="max" value="43"/>
+            <property name="max" value="44"/>
```

Comment:
Is this necessary? What's the offending class?

Reply:
it's the `KafkaAssemblyOperator` class. I guess related to my addition of the storage related clesses.